src/styles/utils/_mixins.scss

```diff
@@ -4,3 +4,50 @@
     #{$_property}: $_toValue;
   }
 }
+
+@mixin Desktop {
+  @media (min-width: 1280px) {
+    @content;
+  }
+}
+  
+@mixin Tablet {
+  @media (min-width: 640px) {
+    @content;
+  }
+}
+
+@mixin top-actions {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 15px;
+  padding-bottom: 20px;
+
+  @include Tablet {
+    padding-top: 20px;
+    padding-bottom: 30px;
+  }
+
+  @include Desktop {
+    padding-top: 40px;
+  }
+}
+
+@mixin use-grid-tablet {
```

Comment:
The mixin 'use-grid-tablet' is declared twice, you should remove the duplicate declaration (lines 46-50).